libs/blog/articles/feature-list/src/lib/category-section-container/category-section-container.component.html

```diff
@@ -3,17 +3,22 @@
   [title]="t(displayName())"
   [link]="{
     displayName: t(displayName()),
-    href: category()
+    href: category(),
   }"
 />
 
 <div class="grid gap-8 md:grid-cols-2 lg:grid-cols-3">
-  @if (articleStore.isFetchArticleListLoading()) {
-    <!--    TODO: add article skleteon when ready-->
-    <!--    <al-article-card-skeleton *alRepeat="take()" class="h-[438px]" />-->
-  } @else {
+  @defer (when !articleStore.isFetchArticleListLoading()) {
     @for (article of articleStore.articles(); track article.slug) {
       <al-article-card [article]="article" [cardType]="cardType()" />
     }
+  } @placeholder {
+    @for (skeleton of skeleonLoaders(); track $index) {
+      @if (cardType() === 'regular') {
+        <al-article-regular-card-skeleton />
+      } @else {
+        <al-article-compact-card-skeleton />
+      }
+    }
   }
```

Comment:
this is bad idea. defer block during ssr fallbacks to `@placeholder`, which ends up rendering skeletons instead of a real content. 
This is a document response from your branch:
![image](https://github.com/HouseOfAngular/angular-love/assets/96125413/1c3559ec-6776-4a02-af65-e5d293ba22cf)
And this is from main:
![image](https://github.com/HouseOfAngular/angular-love/assets/96125413/24f387ef-4e85-4de1-a6a7-463e16222651)